CI test fails on automatic F31 update

```
Error Message
AssertionError: assert 'summary' in {'message': 'Cannot find any applicable policies for koji_build subjects at gating point bodhi_update_push_stable in fedora-31'}
Stacktrace
bodhi_container = DockerContainer(image=bodhi-ci-integration-bodhi/rawhide:latest, id=ec698c9d1b31d539f709ef678adc4d23f01bdb8ce7e5e17cca7bc191a1f36365)
db_container = DockerContainer(image=postgres:latest, id=4e9bc494de58402605b8576f7b03f352c062c4fef4a35023e1da16019de07361)
greenwave_container = DockerContainer(image=bodhi-ci-integration-greenwave:latest, id=9dd6104e4933c37b568501fb4891ca450c412db8d013704a8e91bc860fd5862c)

    def test_updates_query_details(bodhi_container, db_container, greenwave_container):
        """Test getting an update's details with ``bodhi updates query``"""
        # Fetch the last update from the DB
        db_ip = db_container.get_IPv4s()[0]
        query_update = (
            "SELECT "
            "  users.name as username, "
            "  releases.long_name as release, "
            "  updates.* "
            "FROM updates "
            "JOIN users ON updates.user_id = users.id "
            "JOIN releases ON updates.release_id = releases.id "
            "ORDER BY date_submitted DESC LIMIT 1"
        )
        query_comments = (
            "SELECT u.name as username, c.timestamp, c.karma, c.text FROM comments c "
            "JOIN users u ON c.user_id = u.id "
            "WHERE update_id = %s ORDER BY c.timestamp"
        )
        query_karma = (
            "SELECT SUM(comments.karma) as karma FROM comments "
            "JOIN updates ON comments.update_id = updates.id "
            "WHERE update_id = %s"
        )
        query_ct = (
            "SELECT builds.type FROM builds "
            "JOIN updates ON builds.update_id = updates.id "
            "WHERE update_id = %s LIMIT 1"
        )
        query_builds = "SELECT nvr FROM builds WHERE update_id = %s ORDER BY nvr"
        conn = psycopg2.connect("dbname=bodhi2 user=postgres host={}".format(db_ip))
        with conn:
            with conn.cursor() as curs:
                curs.execute(query_update)
                result = curs.fetchone()
                if result is None:
                    pytest.skip("No update in the database")
                update = _db_record_to_munch(curs, result)
                update.comments = []
                curs.execute(query_comments, (update.id, ))
                for record in curs:
                    update.comments.append(_db_record_to_munch(curs, record))
                curs.execute(query_karma, (update.id, ))
                update.karma = _db_record_to_munch(curs, curs.fetchone()).karma
                curs.execute(query_ct, (update.id, ))
                update.content_type = _db_record_to_munch(curs, curs.fetchone()).type
                curs.execute(query_builds, (update.id, ))
                update.builds = [r[0] for r in curs]
        conn.close()
        # Run the command
        result = _run_cli(bodhi_container, ["updates", "query", "--updateid", update.alias])
        assert result.exit_code == 0
        assert "Update ID: {}".format(update.alias) in result.output
        assert "Content Type: {}".format(update.content_type) in result.output
        assert "Release: {}".format(update.release) in result.output
        assert "Status: {}".format(update.status) in result.output
        assert "Type: {}".format(update.type) in result.output
        assert "Severity: {}".format(update.severity) in result.output
        assert "Karma: {}".format(update.karma) in result.output
        expected_autokarma = (
            "Autokarma: {u.autokarma}  [{u.unstable_karma}, {u.stable_karma}]"
        ).format(u=update)
        assert expected_autokarma in result.output
        # If the update doesn't have a request, the CLI does not render the Request: line.
        if update.request:
            assert "Request: {}".format(update.request) in result.output
        # Notes are formatted
        formatted_notes = list(itertools.chain(*[
            textwrap.wrap(line, width=66)
            for line in update.notes.splitlines()
        ]))
        for index, notes_line in enumerate(formatted_notes):
            if index == 0:
                assert "Notes: {}".format(notes_line) in result.output
            else:
                assert "     : {}".format(notes_line) in result.output
        assert "Submitter: {}".format(update.username) in result.output
        expected_submitted = "Submitted: {}".format(
            update.date_submitted.strftime("%Y-%m-%d %H:%M:%S")
        )
        assert expected_submitted in result.output
        for comment in update.comments:
            assert "{user} - {date} (karma {karma})".format(
                user=comment.username,
                date=comment.timestamp.strftime("%Y-%m-%d %H:%M:%S"),
                karma=comment.karma,
            ) in result.output
            # Comments are formatted too
            for index, comment_line in enumerate(textwrap.wrap(comment.text, width=66)):
                assert comment_line in result.output
        assert "1 updates found (1 shown)" in result.output
        # CI Status
        gw_ip = greenwave_container.get_IPv4s()[0]
        greenwave_result = requests.post(
            "http://{}:8080/api/v1.0/decision".format(gw_ip),
            headers={"content-type": "application/json"},
            data=json.dumps({
                "product_version": update.release.lower().replace(' ', '-'),
                "decision_context": (
                    "bodhi_update_push_stable" if update.status == "stable"
                    else "bodhi_update_push_testing"
                ),
                "subject": [
                    {"item": b, "type": "koji_build"}
                    for b in update.builds
                ] + [
                    {"item": update.alias, "type": "bodhi_update"}
                ],
                "verbose": True,
            }),
        ).json()
>       assert "summary" in greenwave_result
E       AssertionError: assert 'summary' in {'message': 'Cannot find any applicable policies for koji_build subjects at gating point bodhi_update_push_stable in fedora-31'}

devel/ci/integration/tests/test_bodhi_cli.py:431: AssertionError
Standard Error
10:19:47.635 __init__.py       DEBUG  exception while getting a value KeyError('RepoTags',) from {'Id': '4e9bc494de58402605b8576f
10:19:47.648 container.py      INFO   running command ['bodhi', 'updates', 'query', '--updateid', 'FEDORA-2019-c2fc735308', '--url', 'http://localhost:8080']
10:19:51.790 container.py      INFO   ================================================================================
     maim-5.5.3-1.fc31
================================================================================
   Update ID: FEDORA-2019-c2fc735308
Content Type: rpm
     Release: Fedora 31
      Status: stable
        Type: unspecified
    Severity: unspecified
       Karma: 0
   Autokarma: True  [-3, 3]
   CI Status: Bodhi failed to send POST request to Greenwave at the following URL "http://greenwave:8080/api/v1.0/decision". The status code was "404". The error was "{'message': 'Cannot find any applicable policies for koji_build subjects at gating point bodhi_update_push_stable in fedora-31'}".
       Notes: Automatic update for maim-5.5.3-1.fc31.
   Submitter: aymenq
   Submitted: 2019-07-27 00:35:19
    Comments: bodhi - 2019-07-27 00:35:19 (karma 0)
              This update was automatically created
              bodhi - 2019-07-27 00:35:19 (karma 0)
              This update's test gating status has been changed to 'waiting'.
              bodhi - 2019-07-27 00:35:20 (karma 0)
              This update's test gating status has been changed to 'ignored'.
              bodhi - 2019-07-27 00:36:07 (karma 0)
              This update can be pushed to stable now if the maintainer wishes
              bodhi - 2019-07-27 00:36:08 (karma 0)
              This update has been submitted for stable by bodhi.

  http://localhost:8080/updates/FEDORA-2019-c2fc735308

1 updates found (1 shown)
10:19:51.863 __init__.py       DEBUG  exception while getting a value KeyError('RepoTags',) from {'Id': '9dd6104e4933c37b568501fb
```

--- a/devel/ci/integration/greenwave/policy.yaml
+++ b/devel/ci/integration/greenwave/policy.yaml
@@ -18,7 +18,6 @@ rules:
   - !PassingTestCaseRule {test_case_name: compose.desktop_terminal, scenario: "fedora.KDE-live-iso.x86_64.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.desktop_terminal, scenario: "fedora.Workstation-live-iso.x86_64.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.install_anaconda_text, scenario: "fedora.universal.x86_64.64bit"}
-  - !PassingTestCaseRule {test_case_name: compose.install_arm_image_deployment_upload, scenario: "fedora.Minimal-raw_xz-raw.xz.arm.ARM"}
   - !PassingTestCaseRule {test_case_name: compose.install_default, scenario: "fedora.Everything-boot-iso.x86_64.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.install_default, scenario: "fedora.Everything-boot-iso.x86_64.uefi"}
   - !PassingTestCaseRule {test_case_name: compose.install_default, scenario: "fedora.KDE-live-iso.x86_64.uefi"}
@@ -40,13 +39,10 @@ rules:
   - !PassingTestCaseRule {test_case_name: compose.install_multi, scenario: "fedora.universal.x86_64.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.install_multi, scenario: "fedora.universal.x86_64.uefi"}
   - !PassingTestCaseRule {test_case_name: compose.install_no_user, scenario: "fedora.KDE-live-iso.x86_64.64bit"}
-  - !PassingTestCaseRule {test_case_name: compose.install_no_user, scenario: "fedora.Workstation-live-iso.x86_64.64bit"}
-  - !PassingTestCaseRule {test_case_name: compose.install_repository_http_graphical, scenario: "fedora.universal.i386.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.install_repository_http_graphical, scenario: "fedora.universal.x86_64.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.install_repository_http_variation, scenario: "fedora.universal.x86_64.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.install_sata, scenario: "fedora.universal.x86_64.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.install_sata, scenario: "fedora.universal.x86_64.uefi"}
-  - !PassingTestCaseRule {test_case_name: compose.install_scsi_updates_img, scenario: "fedora.universal.i386.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.install_scsi_updates_img, scenario: "fedora.universal.x86_64.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.realmd_join_sssd, scenario: "fedora.Server-dvd-iso.x86_64.64bit"}
   - !PassingTestCaseRule {test_case_name: compose.server_cockpit_default, scenario: "fedora.Server-dvd-iso.x86_64.64bit"}
@@ -59,21 +55,20 @@ rules:
 --- !Policy
 id: "taskotron_release_critical_tasks_for_testing"
 product_versions:
+  - fedora-31
   - fedora-30
   - fedora-29
   - fedora-28
-  - fedora-27
-  - fedora-26
 decision_context: bodhi_update_push_testing
 blacklist: []
 subject_type: koji_build
-relevance_value: koji_build
 rules:
-  - !PassingTestCaseRule {test_case_name: dist.rpmdeplint}
   - !RemoteRule {}
+
 --- !Policy
 id: "taskotron_release_critical_tasks_for_stable"
 product_versions:
+  - fedora-31
   - fedora-30
   - fedora-29
   - fedora-28
@@ -82,10 +77,9 @@ product_versions:
 decision_context: bodhi_update_push_stable
 blacklist: []
 subject_type: koji_build
-relevance_value: koji_build
 rules:
-  - !PassingTestCaseRule {test_case_name: dist.rpmdeplint}
   - !RemoteRule {}
+
 --- !Policy
 id: "no_requirements_testing"
 product_versions:
@@ -97,14 +91,13 @@ product_versions:
   - fedora-29-flatpaks
   - fedora-28-modular
   - fedora-28-containers
-  - fedora-28-flatpaks
   - fedora-epel-7
   - fedora-epel-6
 decision_context: bodhi_update_push_testing
 blacklist: []
 subject_type: koji_build
-relevance_value: koji_build
 rules: []
+
 --- !Policy
 id: "no_requirements_for_stable"
 product_versions:
@@ -116,11 +109,38 @@ product_versions:
   - fedora-29-flatpaks
   - fedora-28-modular
   - fedora-28-containers
-  - fedora-28-flatpaks
   - fedora-epel-7
   - fedora-epel-6
 decision_context: bodhi_update_push_stable
 blacklist: []
 subject_type: koji_build
-relevance_value: koji_build
 rules: []
+
+--- !Policy
+# openQA policies
+id: "openqa_release_critical_tasks_for_testing"
+product_versions:
+  - fedora-30
+  - fedora-29
+  - fedora-28
+  - fedora-27
+  - fedora-26
+decision_context: bodhi_update_push_testing
+blacklist: []
+subject_type: bodhi_update
+rules:
+  - !RemoteRule {}
+
+--- !Policy
+id: "openqa_release_critical_tasks_for_stable"
+product_versions:
+  - fedora-30
+  - fedora-29
+  - fedora-28
+  - fedora-27
+  - fedora-26
+decision_context: bodhi_update_push_stable
+blacklist: []
+subject_type: bodhi_update
+rules:
+  - !RemoteRule {}

